utils/api.ts

```diff
@@ -47,6 +47,7 @@ interface TrackComment {
   name: string;
   commenterProfileUrl: string;
   commenterArtworkUrl: string;
+  isNostr: boolean;
```

Comment:
We should also update the rest of this interface with the updated schema coming from catalog. Sample comment response:
```
            {
                "id": 4329,
                "replies": [],
                "isNostr": false,
                "contentId": "42bdf099-9dfb-4400-871f-f6a10fd298e7",
                "msatAmount": 200000,
                "userId": "abc123",
                "createdAt": "2024-01-18T16:02:11.740Z",
                "content": "hi there",
                "commenterArtworkUrl": "https://d12wklypp119aj.cloudfront.net/image/image.jpg",
                "name": "my_alias",
                "title": "Katherines wheel"
            }
```